packages/integrations/solid/package.json

```diff
@@ -35,15 +35,15 @@
     "dev": "astro-scripts dev \"src/**/*.ts\""
   },
   "dependencies": {
-    "vite-plugin-solid": "^2.7.0"
+    "vite-plugin-solid": "^2.8.0"
   },
   "devDependencies": {
     "astro": "workspace:*",
     "astro-scripts": "workspace:*",
     "solid-js": "^1.8.5"
   },
   "peerDependencies": {
-    "solid-js": "^1.4.3"
+    "solid-js": "^1.7.2"
```

Comment:
We could merge the `vite-plugin-solid` upgrade for the vite 5 support, but this peer dep line should be reverted.